05_test_v1aX.sh

```diff
@@ -19,4 +25,4 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -e "DEFAULT_HOSTS_MEMORY=$DEFAULT_HOSTS_MEMORY" \
     -e "CAPI_VERSION=$CAPI_VERSION" \
     -i vm-setup/inventory.ini \
-    -b -vvv vm-setup/v1a2_integration_test.yml
+    -b -vvv vm-setup/v1aX_integration_test.yml -e "VERSION=${ALPHAVERSION}"
```

Comment:
the ansible playbook has the CAPI_VERSION variable, no need for a new one

Reply:
done